doc/stdenv.xml

```diff
@@ -1360,6 +1360,209 @@ in the default system locations.</para>
 
 </section>
 
+<section xml:id="sec-hardening-in-nixpkgs"><title>Hardening in Nixpkgs</title>
+
+<para>There are flags available to harden packages at compile or link-time.
+These can be toggled using the <varname>stdenv.mkDerivation</varname> parameters
+<varname>hardeningDisable</varname> and <varname>hardeningEnable</varname>.
```

Comment:
Why not a single flag `hardening = (true|false)`? Seems strange to have two Boolean arguments that are each other's opposite.